examples/compare-tokens.rs

```diff
@@ -31,14 +31,15 @@ async fn main() -> Result<()> {
         let serialized_pk = (pk,).serialized()?.into_owned();
         let t = Murmur3Partitioner::hash(prepared.compute_partition_key(&serialized_pk)?).value;
 
-        let statement_info = scylla::transport::load_balancing::Statement {
```

Comment:
It would be nice to put the name change of `Statement` to `QueryInfo` in a separate commit. The way it's now it makes it much harder to get acquainted with the new interface because of all the clutter.

Reply:
I have the impression that the added value of extracting it into separate commit would require a disproportionate amount of work now.